README.md

```diff
@@ -0,0 +1,53 @@
+# Flatpak for DOSBox
+
+## Installation
+
+This Flatpak is available on
+[Flathub](https://flathub.org/apps/details/com.dosbox.DOSBox).
+After following the [Flatpak setup guide](https://flatpak.org/setup/),
+you can install it by entering the following command in a terminal:
+
+```bash
+flatpak install --user flathub com.dosbox.DOSBox -y
+```
+
+Once the Flatpak is installed, you can run DOSBox using your desktop environment's
+application launcher.
+
+## Updating
+
+This Flatpak follows the latest stable DOSBox version.
+To update it, run the following command in a terminal:
+
+```bash
+flatpak update
+```
+
+## Limitations
+
+- None known. Please report an issue if you find any.
```

Comment:
There is actually one big limitation that is a feature: This Flatpak is sandboxed and has by-default access only to `xdg-documents` (_Documents_ directory in user home). It is mentioned in the AppData file.